controller/proxy-injector/fake/data/config-proxy.yaml

```diff
@@ -1,66 +1,62 @@
 env:
-- name: LINKERD2_PROXY_LOG
-  value: warn,linkerd2_proxy=info
```

Comment:
Looks like your editor is inserting some extra spaces that changed all the lines. Same for inject-sidecar-container-spec.yaml.

Reply:
eh, thanks. i'll fix that.